components/input/ConfirmationPanelWrapper.tsx

```diff
@@ -22,7 +22,7 @@ const ConfirmationPanelWrapper = <FormFieldValues extends FieldValues>({
         id={name}
         name={name}
         label={label}
-        checked={value}
+        checked={value || false}
```

Comment:
La til defaultvalue til false for å unngå at komponenten går fra uncontrolled til controlled. 

https://stackoverflow.com/questions/47012169/a-component-is-changing-an-uncontrolled-input-of-type-text-to-be-controlled-erro